### DISCUSSION TITLE
Date format : 5/7/2023  => 05/07/2023

### DISCUSSION THREAD

**Discussion by nchaitanyavarma** [wojtekmaj/react-datetime-picker#393] [Q&A] [answered]
HI,
when we select a date between 1 to 9, Ex: 5/7/2023 we are getting in this format , but  we need 05/07/2023 format is it possible 
5/7/2023  => 05/07/2023

**Comment by wojtekmaj** [comment, marked as answer]
Yes, see `format` in documentation.

**Reply by nchaitanyavarma** [reply]
hi, i tried but not getting zero before day 5/7/2023 => 05/07/2023 , we need in the place of 5 we need 05

**Reply by wojtekmaj** [reply]
You tried what?

![image](https://github.com/wojtekmaj/react-datetime-picker/assets/5426427/4b4666dd-a14f-41f2-ae3b-ac13b61c7dc6)
